src/file.ts

```diff
@@ -469,6 +470,8 @@ export enum FileExceptionMessages {
   UPLOAD_MISMATCH = `The uploaded data did not match the data from the server.
     As a precaution, the file has been deleted.
     To be sure the content is the same, you should try uploading the file again.`,
+  MD5_RESUMED_UPLOAD = 'MD5 cannot be used with a resumed resumable upload as MD5 cannot be extended from an existing value',
```

Comment:
Totally nit picky but should be call resuming something else, i.e. continued resumable upload. Resumed resumable upload sounds tongue twisterish.